Boborealmoon/spm_
# Employee own schedule

Code for User Story Employees view their own schedule on a particular date.

Testing completed

Case ID: TC-008
Test Scenario: Employees view their own schedule on a particular date
Pre-conditions: Ensure logged in as Employee

Test steps: 
1. Go to Staff Home page
2. Click on Dropdown to select specified date
3. User should be able to view Schedule indicated on the calendar

Test Data:

Expected result
"2024-09-26"
"2024-08-25"
"2024-08-24"

Should have a WFH event displayed on calendar

Actual results
- Same as expected results

Pass/Fail
- Pass

latest Date of execution
- 2024/09/27
